testsuite/modern-testsuite/src/test/java/org/sonatype/nexus/testsuite/maven/concurrency/MavenHostedSwarmIT.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.testsuite.maven.concurrency;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Callable;
+
+import org.sonatype.nexus.repository.Repository;
+import org.sonatype.nexus.repository.config.Configuration;
+import org.sonatype.nexus.repository.maven.policy.VersionPolicy;
+import org.sonatype.nexus.testsuite.maven.Maven2Client;
+import org.sonatype.nexus.testsuite.maven.concurrency.generators.Generator;
+import org.sonatype.nexus.testsuite.maven.concurrency.generators.XmlGenerator;
+import org.sonatype.nexus.testsuite.maven.concurrency.generators.ZipGenerator;
+import org.sonatype.sisu.goodies.common.ByteSize;
+
+import com.google.common.base.Supplier;
+import com.google.common.base.Suppliers;
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpResponse;
+import org.junit.Test;
+import org.ops4j.pax.exam.spi.reactors.ExamReactorStrategy;
+import org.ops4j.pax.exam.spi.reactors.PerClass;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.sonatype.nexus.testsuite.maven.concurrency.generators.Generator.generatedEntity;
+
+/**
+ * Maven hosted swarm: multiple clients asking, while others are deploying the SAME artifact from a hosted repository.
+ */
+@ExamReactorStrategy(PerClass.class)
+public class MavenHostedSwarmIT
+    extends MavenHotspotITSupport
+{
+  private static final String RELEASE_XML_ARTIFACT_PATH = "groupId/artifactId/1.0/artifactId-1.0.xml";
```

Comment:
Not in all, only half of them (3 out of 6 IT classes). While they have same name and value, I did not consider them worth to move and "pollute" parent class, as originally they were there.